<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Data</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDates">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Dates</string>
   </property>
   <widget class="QRadioButton" name="rbtnData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>All Data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnSelectRanges">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Ranges</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDateTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QDateTimeEdit" name="dateStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>M/d/yyyy hh:mm </string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateEnd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>M/d/yyyy hh:mm </string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblEndDate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From Date:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEditFileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblOutputFile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
